playbooks/development-environments.md

```diff
@@ -39,8 +39,7 @@ Setup tooling is optimized for a default set of tools:
 - `bash` shell
 - [Homebrew](https://brew.sh/) for installing and managing supporting services
 - Docker for containerization
-- [nvm](https://github.com/nvm-sh/nvm) for managing node versions
-- [rbenv](https://github.com/rbenv/rbenv) for managing ruby versions
+- [asdf]([https://github.com/nvm-sh/nvm](https://asdf-vm.com/)) for managing ruby and nodeJS versions
```

Comment:
This seems malformed... maybe more like:
```suggestion
- [asdf](https://asdf-vm.com/) for managing ruby and nodeJS versions
```

Reply:
Ah, this is what trying to use the github editor gets me. Thanks for the catch!